src/utils/string/StringUtils.js

```diff
@@ -64,9 +64,8 @@ const detectStateInCity = (city) => {
  * @returns {String}
  */
 const displayCityName = (city, state) => {
-  if (city === null || city === undefined || city === '') throw new Error('City must be defined');
-  if (state === null || state === undefined || state === '')
-    throw new Error('State must be defined');
+  if (city === null || city === undefined || city === '') return null;
+  if (state === null || state === undefined || state === '') return null;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from throwing an error to returning null when city or state is not defined could potentially introduce bugs if other parts of the code expect an error to be thrown in these cases. Ensure that this change is reflected throughout your codebase.



<!-- This is an auto-generated comment by OSS CodeRabbit -->